doc/changelog.rst

```diff
@@ -60,6 +68,12 @@ Guide
 General/miscellaneous
 ~~~~~~~~~~~~~~~~~~~~~
 
+- MultiLanguageModelForm: Avoid partially/empty translation objects
+  - Delete untranslated objects from database
+  - Only set translation object to database if it is translated
+  - Ensure required fields if language is partially translated
```

Comment:
These won't render properly when viewing the file in html. I think you have to add more spaces or something